tests/unittest/cloud_governance/common/clouds/aws/athena/test_athena_operations.py

```diff
@@ -0,0 +1,15 @@
+from cloud_governance.common.clouds.aws.athena.abstract_athena_operations import AbstractAthenaOperations
```

Comment:
do we need to import the abstract class ?

Reply:
NO